modules/300-prometheus/docs/README_RU.md

```diff
@@ -74,17 +74,23 @@ Deckhouse поддерживает отправку алертов с помощ
 
 <img src="../../images/300-prometheus/prometheus_monitoring.svg">
 
+WIP:  новая схема для единого  датасорса
```

Comment:
@pashcovich Can we remove the WIP here?

Reply:
I think only if we point the PMA to a new scheme